rdmo/questions/managers.py

```diff
@@ -18,6 +18,34 @@ def filter_catalog(self, catalog):
     def prefetch_elements(self):
         return self.prefetch_related(*self.model.prefetch_lookups)
 
+    def filter_for_user(self, user):
```

Comment:
Good idea, I was wondering why we did not do this earlier, it seems the only difference is the ordering. Since ordering is not free (on the database), we should may move that back to the view?

Reply:
On the other hand, the non filtering is just in one view, which we will remove anyway. Just keep it here.